### PR TITLE
User info endpoints implemented

### PR DESCRIPTION
**Pull request by luisferreira32**
++ update user info and get user info endpoints
++ tweaks on database read and endpoint models

### CODE REVIEW

File: services/user/justfile [hunk 1]
```diff
@@ -2,3 +2,6 @@ set dotenv-load := false
 
 proto:
     python -m grpc_tools.protoc -I. --python_out=. --pyi_out=. --grpc_python_out=. user.proto
+
+personal:
```
Comment (antoniomf97) [inline services/user/justfile]: Please remove or change label

**Review by antoniomf97 — CHANGES_REQUESTED** [review]
Just a quick request.

File: services/user/justfile
Comment (bgonilho) [inline services/user/justfile:1]: only use "personal" when working locally

File: services/user/justfile (hunk 1 above)
Comment (bgonilho) [inline services/user/justfile]: Only use "personal" when working locally

**Review by antoniomf97 — APPROVED** [review]
Better.

File: services/user/service.py [hunk 2]
```diff
@@ -225,14 +226,39 @@ def SignUp(self, req: model.SignupRequest) -> None:
...
+        # TODO: check against a DB admin if we have permissions to check this up
+        return []
+
+    def UpdateUserInfo(self, req: model.UpdateUserReq) -> model.UserInfo:
+        # NOTE: this assumes the server already verified them, being the entry point of the request
+        claims = jwt.get_unverified_claims(req.access_token)
+        # NOTE: always force you can only update your user entry
+        req.user_id = claims["sub"]
+
+        # NOTE: read before write, do not do this if you're a sane person
```
Comment (luisferreira32) [inline services/user/service.py:257]: 👀

**Review by bgonilho — APPROVED** [review]

**Pull request merged**